content/dotnet-template-azure-iot-edge-module/CSharp/ModuleBackgroundService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.Azure.Devices.Client;
+using Microsoft.Azure.Devices.Client.Transport.Mqtt;
+using System.Text;
+
+namespace SampleModule
+{
+    internal class ModuleBackgroundService : BackgroundService
+    {
+        private int counter;
```

Comment:
Consistent naming style.

```suggestion
        private int _counter;
```